### PR TITLE
Booyah 9/setup testing engine

### PR DESCRIPTION
**Pull request by marceloribeiro**

### CODE REVIEW

File: src/tests/lib/controllers/application_controller_test.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+from lib.controllers.application_controller import ApplicationController
+import io
+
+class HomeController(ApplicationController):
+    def index(self):
+        print('inside', True)
```
Comment (klaus-trabalhando) [inline src/tests/lib/controllers/application_controller_test.py]: is still required this print?

File: src/application.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 read_dotenv(dotenv_path)
 print('Spinning up environment [' + os.getenv('BOOYAH_ENV') + ']')
 
-from lib.application.application_router import ApplicationRouter
+from lib.router.application_router import ApplicationRouter
```
Comment (klaus-trabalhando) [inline src/application.py:8]: I did not see any place that set the BOOYAH_ENV to test.

File: src/tests/lib/controllers/application_controller_test.py (hunk 1 above)
Comment (marceloribeiro) [reply src/tests/lib/controllers/application_controller_test.py]: @klaus-trabalhando  this should be solved now

File: src/application.py (hunk 2 above)
Comment (marceloribeiro) [reply src/application.py:8]: @klaus-trabalhando I addressed this by adding the env loader to test conf

**Pull request merged**